src/libraries/Native/Unix/System.Native/pal_io.c

```diff
@@ -921,7 +921,16 @@ int32_t SystemNative_Poll(PollEvent* pollEvents, uint32_t eventCount, int32_t mi
 
 
     int useStackBuffer = bufferSize <= 2048;
```

Comment:
```suggestion
    int useStackBuffer = bufferSize <= sizeof(stackBuffer);
```

... and move it after stackBuffer.

Reply:
Thanks! Done.